ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -85,37 +100,3 @@
     owner: root
     group: root
     mode: 0770
-
-- name: Check for encrypted drive in fstab
-  shell: grep '{{ encrypted_root }} ecryptfs' /etc/fstab
-  register: fstab_contents
-  failed_when: false
-  changed_when: false
-  always_run: true
-
-- name: Register mount configuration
-  sudo: yes
-  shell: "grep '{{ encrypted_root }} ecryptfs' /etc/mtab"
-  register: mount_conf
-  changed_when: false
-  always_run: true
-
-- name: add user, noauto to mount conf
-  shell: "echo {{ mount_conf.stdout }} | sed -e 's/rw/user,noauto,rw/'"
-  when: "'noauto' not in mount_conf.stdout"
-  register: updated_mount_conf
-
-- name: Create entry in fstab
-  sudo: yes
-  lineinfile:
-    dest: "/etc/fstab"
-    regexp: "^{{ encrypted_root }}"
-    backup: yes
-    line: "{{ updated_mount_conf.stdout if not 'skipped' in updated_mount_conf else mount_conf.stdout }}"
-  when: "'{{ encrypted_root }}' not in fstab_contents.stdout"
-
-- name: Verify that entry is in fstab
-  sudo: yes
-  shell: "grep -q '^{{ encrypted_root }}\\s*{{ encrypted_root }}\\s*ecryptfs' /etc/fstab"
-  always_run: True
-  changed_when: False
```

Comment:
Removed because it is not possible to mount the encrypted drive on boot (obviously), and the entry in fstab causes the boot to hang. I have removed the extra entries from fstab on prod and will do it soon on softlayer.